wom/src/main/scala/wdl/WdlExpression.scala

```diff
@@ -255,18 +255,17 @@ object WdlWomExpression {
 
     for {
       resolvedVariables <- expression.wdlExpression.variableReferences.toList traverse resolveVariable
-    } yield GraphNodeInputExpression(name, expression, resolvedVariables.toMap)
```

Comment:
FMI I presume `GraphNodeInputExpression` is going away because we do input expression evaluation outside the main graph node now?

Reply:
This was only really used by InstantiatedExpression which I removed to only have ExpressionNodes so that's why it disappeared